angelspetrochem/data.lua

```diff
@@ -28,8 +28,10 @@ angelsmods.trigger.enableacids = settings.startup["angels-enable-acids"].value
 angelsmods.trigger.slag = true
 angelsmods.trigger.salt_production = true
 angelsmods.trigger.water_greenyellow_waste = angelsmods.trigger.enableacids or (angelsmods.functions.is_special_vanilla() == false)
-angelsmods.trigger.water_green_waste = angelsmods.trigger.enableacids
-angelsmods.trigger.water_red_waste = false
```

Comment:
We usually put a default value for each trigger in the mod that 'manages' the trigger...
In this case, I would assume red waste water does not have a trigger anymore?
However it seems bio is setting the trigger to true... so here it should either also be true, or at least be set to a default value of false, preventing it from being nil?

Reply:
Waste waters are added by Refining. There is still a trigger for red waste water. It's initialized in `angelsrefining\data.lua`.